draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -24,28 +24,26 @@ author:
 normative:
   RFC3986: RFC3986
   RFC7800: RFC7800
-  RFC6755: RFC6755
   RFC7591: RFC7591
   RFC7519: RFC7519
   RFC8414: RFC8414
   RFC8725: RFC8725
+  RFC9110: RFC9110
+  RFC9112: RFC9112
 informative:
   RFC6749: RFC6749
-  RFC7521: RFC7521
   ARF:
   	title: "The European Digital Identity Wallet Architecture and Reference Framework"
 
 
 --- abstract
 
-This specification defines a new method of client authentication for OAuth 2.0 {{RFC6749}} by extending the approach defined in {{RFC7521}}. This new method enables client deployments that are traditionally viewed as public clients to be able to authenticate with the authorization server through an attestation based authentication scheme.
+This specification defines an extension to the OAuth 2 protocol as defined in {{RFC6749}} which enables a Client Instance to include an attestation in interactions with an Authorization Server or a Resource Server. This new method enables Client Instances involved in a client deployment that is traditionally viewed as a public client, to be able to provide an attestation in order to authenticate.
```

Comment:
```suggestion
This specification defines an extension to the OAuth 2 protocol as defined in {{RFC6749}} which enables a Client Instance to include an attestation in interactions with an Authorization Server or a Resource Server. This new method enables Client Instances involved in a client deployment that is traditionally viewed as a public client, to be able to authenticate.
```

Reply:
How about that? If we rename the draft we may rename attestation, but until then I would keep it
```suggestion
This specification defines an extension to the OAuth 2 protocol as defined in {{RFC6749}} which enables a Client Instance to include a key-bound attestation in interactions with an Authorization Server or a Resource Server. This new method enables Client Instances involved in a client deployment that is traditionally viewed as a public client, to be able to utilize this key-bound attestation to authenticate.
```

Reply:
I like your proposal.